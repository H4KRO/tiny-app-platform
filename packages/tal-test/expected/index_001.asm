main()
  entry:
    Literal           0
    Literal           1
    Literal           1
    MakeArray
    Index
