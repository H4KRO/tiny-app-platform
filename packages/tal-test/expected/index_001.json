[
  {
    "kind": "Index",
    "index": 0,
    "value": {
      "kind": "Array",
      "value": [
        1
      ]
    }
  }
]